exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -480,6 +480,9 @@ private void generateOptionsDescriptionJSFile() throws IOException {
     OptionManager optionManager = this.drillbit.getContext().getOptionManager();
     OptionList publicOptions = optionManager.getPublicOptionList();
     List<OptionValue> options = new ArrayList<>(publicOptions);
+    //Add internal options
```

Comment:
Do we need internal options on Web UI?

Reply:
Since there is a WebUI interface for internal options that has the same layout, I thought it should be added. Part of this exercise is to make it easier for Doc team to also generate web-site content that is consistent.

Reply:
If we adding these options to the Web UI, maybe there should be some quick filter? How we'll be able to distinguish such options?

Reply:
Oh. The webpages are different.
The public options are accessible via `http://<drillHost>/options`
The internal options are accessible via `http://<drillHost>/internal_options` , but there is no visible link. Users (developers/admin) need to be aware of it. So, it is not easily accessible.

Reply:
Just wondering how developers/admin will be aware of it? I first heard about it from you :)

Reply:
I guess I really dug into the Freemarker pages to have discovered this. 😄 
Not sure where (if at all) this was documented. But since the page exists, it makes sense to have the descriptions available as well.